src/lm_buddy/integrations/wandb/run_config.py

```diff
@@ -21,12 +21,11 @@ class WandbRunConfig(BaseLMBuddyConfig):
     if it is not provided.
     """
 
-    __match_args__ = ("run_id", "name", "project", "run_group", "entity")
+    __match_args__ = ("id", "project", "group", "entity")
```

Comment:
I've renamed these fields to exactly match the names of the fields in `wandb.init`. This is so we can dump a dict of this dataclass directly as `**kwargs` when calling that.